scripts/check-esm-imports.js

```diff
@@ -40,11 +40,11 @@ function readLines(filePath) {
 }
 
 function isLocalImport(row) {
-  return row.includes("from '.") || row.includes('from ".')
+  return row.match(/from ['"]\./)
```

Comment:
Shouldn't this be `test` instead of `match`?

Reply:
nice catch!

changed to use RegExp#test.